### DISCUSSION TITLE
Package version issues with @parcel/transform-babel@next

### DISCUSSION THREAD

**Discussion by webstrand** [parcel-bundler/parcel#6387] [General] [answered]
I'm using the babel transform, but have run into issues with its dependency `@parcel/babylon-walk`. I've added a dependency like
```json
{
  "dependencies": {
    "@parcel/transformer-babel": "^2.0.0-beta.3.1"
  }
}
```
but `@parcel/transformer-babel` doesn't describe its dependency on `@parcel/babylon-walk`.
So I added it, manually, to my project
```json
{
  "dependencies": {
    "@parcel/transformer-babel": "^2.0.0-beta.3.1",
    "@parcel/babylon-walk": "^2.0.0-beta.3.1"
  }
}
```
Which has caused me no end of frustration, since `2.0.0-nightly.1765` is the `latest` tag on NPM and is considered "newer" than `2.0.0-beta.3.1` (which is the `next` tag on NPM).

I'm not sure where the issue lies. I've fixed by pinning the dependency to `"@parcel/babylon-walk": "2.0.0-beta.3.1"` but maybe this is an issue that can be fixed on the packaging end?

**Comment by mischnic** [comment, marked as answer]
-> https://github.com/parcel-bundler/parcel/pull/6388
